server/lib/cron-scheduler/save-document.ts

```diff
@@ -31,7 +31,7 @@ export class SaveDocument {
       const createDocumentObject = this.createDocument(doc, indexCreation, mapping);
       const response = await this.esClientInternalUser.bulk(createDocumentObject);
       log(this.logPath, `Response of create new document ${JSON.stringify(response)}`, 'debug');
-      // await this.checkIndexPatternAndCreateIfNotExists(index);
+      await this.checkIndexPatternAndCreateIfNotExists(index);
```

Comment:
This creates an index pattern through the backend. We want to create it in the frontend.